spring-cloud-stream-binders/spring-cloud-stream-binder-rabbit/src/test/java/org/springframework/cloud/stream/binder/rabbit/RabbitBinderTests.java

```diff
@@ -154,7 +150,7 @@ public void testConsumerProperties() throws Exception {
 		assertEquals(1000L, TestUtils.getPropertyValue(retry, "retryOperations.backOffPolicy.initialInterval"));
 		assertEquals(10000L, TestUtils.getPropertyValue(retry, "retryOperations.backOffPolicy.maxInterval"));
 		assertEquals(2.0, TestUtils.getPropertyValue(retry, "retryOperations.backOffPolicy.multiplier"));
-		binder.unbindConsumers("props.0");
+		binder.unbindConsumers("props.0", "default");
```

Comment:
Good point. We should probably have the same approach with a final superclass method that always ensures "default" is a replacement for either `null` or empty String on unbind as well.